src/blog/2018-10-5/mutation-switching.pug

```diff
@@ -0,0 +1,161 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        
+        We are very happy with Stryker's new friends! One of those new friends is Stryker4s(cala). 
+        Scala developers can now use mutation testing to improve their tests! 
+        Creating Stryker4s is a lot of fun, with a ton of challenges. One of those challenges is the Scala compiler itself.
+        We all know it's not the [fastest in its kind](https://scala-ci.typesafe.com/grafana/dashboard/db/scala-benchmark?from=1429139130109&to=1492531027936h&orgId=1&var-branch=2.11.x&var-branch=2.12.x&var-branch=2.13.x&var-source=scala&var-bench=HotScalacBenchmark.compile&var-host=scalabench@scalabench@). 
+        One of the main goals of Stryker is to be fast. This means that we need to come up with an intelligent 
+        way to introduce the mutants into the source code.
+        
+        ## Attempt one: Compiling each mutation
+        
+        One way of introducing mutants to a codebase is mutating one statement, compile to code, run tests, gather the results and repeat. 
```

Comment:
Shouldn't this be ` compile the code, run the tests,`  instead of `compile to code, run tests`?

Reply:
Good find!